frameworks/Java/voovan/benchmark_config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "framework": "voovan",
+  "framework": "voovan-mvc",
```

Comment:
I'm confused about this name change. If the framework is called `voovan`, we'd prefer this to match the directory `Java/voovan` but you could change the `display_name` attribute to say `voovan-mvc`. If the framework name is changing, then you can change the directory to `voovan-mvc` as well.

Reply:
I agree with what @nbrady-techempower talks about.

If the lib calls `voovan` in Java ecosystem, please keep the origin name. Don't confuse the other people who will contribute to this project in the future.

Reply:
ok i will fixed it